src/Elmish/Enzyme/Foreign.purs

```diff
@@ -72,96 +78,96 @@ mountComponent def = do
 -- | HTML within the given `ReactElement` is rendered, but not HTML within
 -- | nested components. This does not cause a headless browser to spin up.
 -- | See https://enzymejs.github.io/enzyme/docs/api/shallow.html for more info.
-shallow :: forall m. MonadEffect m => ReactElement -> m ElementWrapper
+shallow :: forall m. MonadEffect m => ReactElement -> m (Wrapper SingleNode)
 shallow =
   liftEffect <<< runEffectFn1 shallow_
 
 -- | A convenience function for creating a shallow wrapper from a `ComponentDef`
 -- | rather than a `ReactElement`. This also adds a `delay (Milliseconds 0.0)`
 -- | to allow any `Effect`s in `def.init` to run.
-shallowComponent :: forall m msg state. MonadAff m => ComponentDef msg state -> m ElementWrapper
+shallowComponent :: forall m msg state. MonadAff m => ComponentDef msg state -> m (Wrapper SingleNode)
```

Comment:
I wonder if we need these. From what I understand about how they work, they seem kind of useless, right? And we don't expose monadic versions of them from `Elmish.Enzyme` either.

Reply:
Yeah, I'm fine with removing it 👍

Reply:
Removed